legacy/i18n/locales/en-US.json

```
@@ -372,6 +372,7 @@
   "components.walletselection.walletselectionscreen.header": "My wallets",
   "components.walletselection.walletselectionscreen.stakeDashboardButton": "Stake Dashboard",
   "components.walletselection.walletselectionscreen.loadingWallet": "Loading wallet",
+  "components.walletselection.walletselectionscreen.supportTicketLink": "GET HELP ON YOROI HELPDESK",
```

I'd suggest capitalizing on the view only.
```suggestion
  "components.walletselection.walletselectionscreen.supportTicketLink": "Get help on Yoroi helpdesk",
```